### PR TITLE
Add BitBuffer package.

### PR DESCRIPTION
**Pull request by BinaryProvider**

### CODE REVIEW

File: _data/packages.yml [hunk 1]
```diff
@@ -426,7 +426,30 @@
...
+    ```ts
+    import BitBuffer from "rbx-bitbuffer";
+    // Save data
+    const buffer = BitBuffer.create();
+    buffer.writeString('Example data');
+    buffer.writeBool(false);
+    const data = buffer:toBase64();
+    // Load data
+    const buffer = BitBuffer.create();
+    buffer:fromBase64(data);
```
Comment (osyrisrblx) [inline _data/packages.yml]: This isn't valid TS. Oops?

Comment (BinaryProvider) [reply _data/packages.yml]: Sorry, it's not clear enough :)
It's not meant to be used in the same scope. I'll update to make it clearer.

Thanks for the feedback!

Comment (osyrisrblx) [reply _data/packages.yml]: Actually, I meant that `buffer:toBase64();` isn't valid TS. I think you meant `buffer.toBase64();`

Comment (BinaryProvider) [reply _data/packages.yml]: Omg.. how could I miss that. Copy Paste typo, sorry about that. It should be corrected now.

**Pull request merged**